src/modules/auth/credentials/CredentialsTenantsCard.js

```diff
@@ -200,6 +201,21 @@ function CredentialsTenantsCard(props) {
 										)
 									} else { return null }
 								})}
+								{(count > limit) ?
+									<>
+										<DropdownItem divider />
+										<DropdownItem
+											onClick={() => {
+												setLimit(limit + 5);
+												toggleDropdown(true);
```

Comment:
btw `togglerDropdown` cosnt does not have any parameter to pass as a prop, since its by default empty, so `togglerDropdown(true)` is invalid

```
const toggleDropdown = () => setDropdownOpen(prevState => !prevState);
```